sdcm/cluster.py

```diff
@@ -2147,6 +2148,27 @@ def offline_install_scylla(self, unified_package, nonroot):
             """)
             self.remoter.run('sudo bash -cxe "%s"' % install_cmds)
 
+    def web_install_scylla(self, scylla_version: Optional[str] = None) -> None:
+        """
+        Install scylla by Scylla Web Installer Script. Try to use install the latest
+        unstable build for the test branch, generally it should install same version
+        as the unstable scylla_repo.
+
+        https://github.com/scylladb/scylla-web-install
+        """
+
+        # Try to get the major version from the branch name, it will only be used when scylla_version isn't assigned.
+        # It can be switched to RELEASE_BRANCH from upstream job
+        git_branch = os.environ.get('GIT_BRANCH')  # origin/branch-4.5
+        self.log.debug("scylla_version: %s, git_branch: %s", scylla_version, git_branch)
+
+        if match := re.match(r'\D*(\d+.\d+)', scylla_version or git_branch):
```

Comment:
`.` here matches any char. You need to escape it to match the dot exactly (`\.`)

Reply:
Good catch!fixed.